packages/runtime/src/next/api-handler/handlers/manifest.ts

```diff
@@ -32,7 +33,8 @@ export default async function handler(
     clientSideNavigation: true,
     elementFromPoint: false,
     customBreakpoints: true,
-    unstable_siteVersions,
+    versioning,
+    unstable_siteVersions: versioning,
```

Comment:
Can we get rid of this field? I'm guessing the plan is to get rid of this field as soon as we update our app so that it reads `versioning` instead?